huaweicloud/services/acceptance/dms/data_source_huaweicloud_dms_rocketmq_consumer_groups_test.go

```diff
@@ -0,0 +1,122 @@
+package dms
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDmsRocketMQConsumerGroups_basic(t *testing.T) {
+	name := acceptance.RandomAccResourceName()
+	dataSourceName := "data.huaweicloud_dms_rocketmq_consumer_groups.test"
+	dc := acceptance.InitDataSourceCheck(dataSourceName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDmsRocketMQSearchConsumerGroups(name),
+				Check: resource.ComposeTestCheckFunc(
+					dc.CheckResourceExists(),
+					resource.TestCheckResourceAttrSet(dataSourceName, "groups.#"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "groups.0.name"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "groups.0.enabled"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "groups.0.broadcast"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "groups.0.retry_max_times"),
+					resource.TestCheckOutput("name_filter_is_useful", "true"),
+					resource.TestCheckOutput("enabled_filter_is_useful", "true"),
+					resource.TestCheckOutput("broadcast_filter_is_useful", "true"),
+					resource.TestCheckOutput("retry_max_times_filter_is_useful", "true"),
+				),
+			},
+		},
+	})
+}
+
+func testAccDmsRocketMQSearchConsumerGroups(name string) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "huaweicloud_dms_rocketmq_consumer_group" "test" {
+  instance_id = huaweicloud_dms_rocketmq_instance.test.id
+  
+  brokers = [
+    "broker-0"
+  ]
+  
+  name            = "%[2]s"
+  retry_max_times = "3"
```

Comment:
It's not recommend to set int parameter to **string** although it can work.

Reply:
done